vlib/v/tests/tmpl/if_cond.txt

```diff
@@ -0,0 +1,13 @@
+aaa
+@if cond
+bbb
+@else
+zzz
+@end
+ccc
+
+aaa
+@if cond
+bbb
+@end
+ccc
```

Comment:
I think that it would be good, to add a test, that exercises the @else branch as well, because there were changes that affected it in the parser code.

Also please, make it use separate `tmpl_if_cond_test.v` test file. The `tmpl_test.v` file is getting very long, and hard to follow.

Reply:
Alright. 👍🏻